SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -0,0 +1,120 @@
+# [Map travel](https://docs.unrealengine.com/en-us/Gameplay/Networking/Travelling)
+
+> Note: This doc is intended for advanced users only.
+
+## `APlayerController::ClientTravel`
+
+### In native Unreal
+`ClientTravel` is the process of changing which [map (or Level - see Unreal documentation)](http://api.unrealengine.com/INT/Shared/Glossary/index.html#l) a client currently has loaded.
+
+### User guide
+In the GDK you can use `ClientTravel` to move a client-worker from an offline state to a connected state, whether that be connected to a local deployment or a cloud deployment. Alternatively, you can move a connected client-worker to an offline state. You can also change which SpatialOS deployment the client-worker is connected to. 
+
+Always make sure your client-worker(s) have the same map loaded as the one which the server-worker(s) are running in your deployment. 
+
+#### Using Receptionist
+The Receptionist is a SpatialOS service which allows you to connect to a deployment via a host and port.
```

Comment:
Should we add a section that these ip and port can be specified in the command line like in native unreal, and the receptionist will connect automatically.